### PR TITLE
[wix-ui-tpa]: publish storybook

### PR DESCRIPTION
**Pull request by argshook**
This PR adds `storybook:release` to run after `test9` is successful

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "test6:sanity": "lerna run --scope wix-style-react test6:sanity",
     "test7:performance": "lerna run --scope wix-style-react test7:performance",
     "test8:chromatic": "lerna run --scope wix-style-react-test-runtime test",
-    "test9": "lerna run build --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa",
+    "test9": "lerna run build --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa && lerna run storybook:release --scope wix-ui-tpa",
```
Comment (jonathanadler) [inline package.json:17]: I would run them in parallel or at least put the storybook:release first, so that it will run even when tests will fail

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6736

File: package.json (hunk 1 above)
Comment (argshook) [inline package.json:17]:
```suggestion
    "test9": "lerna run build --scope wix-ui-tpa && lerna run storybook:release --scope wix-ui-tpa && lerna run test --scope wix-ui-tpa",
```

i'd think maybe if tests fail then storybook is falsy and shouldn't be published?
but if that's ok, then you can commit this suggestion and merge

Comment (jonathanadler) [reply package.json:17]: but that's what happens with wsr as well. Janet is parallel

**Pull request merged**